packages/permission-controller/src/PermissionController.ts

```diff
@@ -1717,6 +1719,27 @@ export class PermissionController<
     },
   ): void {
     const { allowedCaveats, validator } = specification;
+
+    if (
```

Comment:
Nit: In the JSDoc comment for this method, there is a list of things that it's responsible for validating. We should add an entry for the subject type